deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -55,77 +55,77 @@ class TrainTestLabelDrift(TrainTestCheck):
 
     Parameters
     ----------
-    alternative_label_measurements : List[Dict[str, Any]], default: None
-        List of measurements. Replaces the default deepchecks measurements.
-        Each measurement is dictionary with keys 'name' (str), 'method' (Callable) and 'is_continuous' (bool),
+    alternative_label_properties : List[Dict[str, Any]], default: None
+        List of properties. Replaces the default deepchecks properties.
+        Each property is dictionary with keys 'name' (str), 'method' (Callable) and 'value_type' (str),
```

Comment:
We should explain value types and their meaning

Reply:
in the user guide